### ISSUE TITLE
PROV support broken

### ISSUE THREAD

**Issue by dakoop** [VisTrails/VisTrails#1075]
When we modified the structure of the execution provenance with respect to `machine` objects, this seems to have broken PROV support. For example, trying exporting the terminator example to PROV produces the following traceback:

```
Traceback (most recent call last):
  File "VisTrails 2.2.app/Contents/Resources/lib/python2.7/vistrails/gui/vistrails_window.py", line 1937, in method
    getattr(obj, method_name)()
  File "VisTrails 2.2.app/Contents/Resources/lib/python2.7/vistrails/gui/vistrail_view.py", line 1062, in save_prov
    self.controller.write_prov(locator)
  File "VisTrails 2.2.app/Contents/Resources/lib/python2.7/vistrails/gui/vistrail_controller.py", line 1132, in write_prov
    locator.save_as(prov_document)
  File "VisTrails 2.2.app/Contents/Resources/lib/python2.7/vistrails/core/db/locator.py", line 158, in save_as
    obj = _XMLFileLocator.save(self, obj, True, version)
  File "VisTrails 2.2.app/Contents/Resources/lib/python2.7/vistrails/db/services/locator.py", line 511, in save
    obj = io.save_to_xml(obj, self._name, version)
  File "VisTrails 2.2.app/Contents/Resources/lib/python2.7/vistrails/db/services/io.py", line 571, in save_to_xml
    return save_prov_to_xml(obj, filename, version)
  File "VisTrails 2.2.app/Contents/Resources/lib/python2.7/vistrails/db/services/io.py", line 1370, in save_prov_to_xml
    prov_document.log)
  File "VisTrails 2.2.app/Contents/Resources/lib/python2.7/vistrails/db/services/prov.py", line 493, in create_prov
    for machine in log._db_machines:
AttributeError: 'Log' object has no attribute '_db_machines'
```

**Comment by remram44** [comment]
Like #1076, probably because of the changes on [`logger-rewrite`](https://github.com/VisTrails/VisTrails/compare/27d7aa484bb0d71e521374ce32a6797aa1660664...7216d0a52db8e521597a5fa9648c6963be0fd27b).

`machine` is no longer on `log`, it's on `workflow_exec`.

**Comment by rexissimus** [comment]
Fixed by 6dc172f0aad39c7c7e23d3e3a3142dffa32290e1
